cpp/roaring64map.hh

```diff
@@ -679,7 +679,7 @@ class Roaring64Map {
         return std::accumulate(
             roarings.cbegin(), roarings.cend(),
             sizeof(uint64_t) + roarings.size() * sizeof(uint32_t),
-            [=](uint64_t previous,
+            [=](size_t previous,
```

Comment:
I think is correct to be a uint64_t, size_t can be 32 bit when compiled for a 32 bit processor.